pynvim/_version.py

```diff
@@ -0,0 +1,13 @@
+"""Specifies pynvim version."""
+# pylint: disable=consider-using-f-string
+
+from types import SimpleNamespace
+
+# see also setup.py
+VERSION = SimpleNamespace(major=0, minor=5, patch=0, prerelease="dev0")
```

Comment:
Do we need to mention this in our release docs: https://github.com/neovim/pynvim#release

Reply:
Oh I see, I missed that, will fix this.